app/models/guide.rb

```diff
@@ -58,7 +58,7 @@ def done_for?(user)
   end
 
   def import_from_json!(json)
-    self.assign_attributes json.except('exercises', 'language', 'id_format', 'id', 'teacher_info', 'collaborators')
+    self.assign_attributes json.except('exercises', 'language', 'id_format', 'id', 'teacher_info', 'collaborators', 'private')
```

Comment:
We must stop doing this. See https://github.com/mumuki/mumuki-laboratory/issues/1161